JetSet/JetSet/Models/Todo.swift

```diff
@@ -12,4 +12,10 @@ struct Todo: Identifiable, Codable {
     let description: String
     let action: String
     var done: Bool
+    
```

Comment:
After reading into how CodingKeys is meant to work, it promotes better conformance to the Codable class than the Todo struct otherwise would on its own. Additionally, it allows for much easier encoding and decoding of JSON data through the enumerator map keys specified below. 

I think this is especially useful for future components that will be added to the application, as CodingKeys will allow for the encoding and decoding of user-generated JSON checklist data.

Reply:
Yeah so like by default when you have a model conform to `Codable`, `CodingKeys` are actually automatically generated at compile time to match the names of all the variables present for the class.

So by default, the compiler expects your JSON data to have an `id` key/value pair. A couple things here:
* In `Todo.swift` you define `let id = UUID()` which previously was raising a compiler error basically saying it's going to get overridden when decoding from data. This is because the compiler is expecting that when you decode form data, the `UUID()` you set here is just going to be overwritten by the data present from your decoding source (JSON).
* By specifying a `CodingKeys` enum _inside_ the scope of `Todo` (or any `Codable` model type), you're basically overriding the automatically generating compile time keys to say "These are the only key names I expect to decode from/encode to. This makes the compiler warning mentioned above go away!

Reply:
Because your `CodingKeys` specifically does _not_ list `case id`, the compiler knows that that property won't be overwritten during decoding/encoding